_includes/azerothcore/sidebar.html

```diff
@@ -49,9 +49,4 @@
         <li><a href="/wiki/Waypoints-Information">Information about waypoints</a></li>
         <li><a href="/wiki/How-to-obtain-free-records">How to obtain free records</a></li>
     </ul>
-    <b>Archive</b>
-    <ul>
-        <li><a target="_blank" href="http://www.azerothcore.org/pages/sunwell.pl/">Original SC Description</a></li>
```

Comment:
why we should remove this? I would keep it

Reply:
I remove it because it serves little to no purpose

Reply:
for me it's useful

Reply:
I agree with @Helias